src/Common/AzurePowerShell.cs

```diff
@@ -22,7 +22,7 @@ public class AzurePowerShell
     {
         public const string AssemblyCompany = "Microsoft";
 
-        public const string AssemblyProduct = "Microsoft Azure Powershell";
+        public const string AssemblyProduct = "Microsoft Azure PowerShell";
```

Comment:
Did we talk about keeping the wrong spelling as-is, because we don't want the telemetry data to have both spelling?

Reply:
It's a different one. Product name is used for branding.